### DISCUSSION TITLE
Error calling other activation function arguments?

### DISCUSSION THREAD

**Discussion by jmenville11** [Project-MONAI/MONAI#2845] [Q&A]
Hello,

In playing around with the 3D UNet Model, I see that sigmoid and softmax are written as booleans directly into the code, but there is also an option to call other activation functions. Can you point to the documentation that lists the arguments for these other activation codes? 

I've been able to use "sigmoid=True", "softmax=True", and "other_act=torch.tanh", but not PReLU, Softmin, or LogSoftmax. For example:

Abbreviated Code:
      import torch
      import torch.nn as nn 
      etc.

      dice_metric = DiceMetric(include_background=True, reduction="mean")
      dice_metric_batch = DiceMetric(include_background=True, reduction="mean_batch")
      post_trans = Compose(
          [ToTensor(), Activations(other_act=nn.LogSoftmax), AsDiscrete(threshold_values=True)]

Error:
File "ICH_UNet_Model_Experiment_Parameters.py", line 216, in <module>
    [ToTensor(), Activations(other_act=nn.LogSoftmax), AsDiscrete(threshold_values=True)]
TypeError: __init__() got an unexpected keyword argument 'other_act'

**Comment by rijobro** [comment]
Hi @jmenville11 , the argument is `other`, not `other_act`.
